### ISSUE TITLE
Exogenise/endogenise with several shocks

### ISSUE THREAD

**Issue by AndreyDO** [IRIS-Solutions-Team/IRIS-Toolbox#240]
Hi, guys!

Is the option of weighting still working with endogenising with several shocks via 'simulate' command?
(Basically, the stuff discussed here: https://groups.google.com/forum/#!topic/iris-toolbox/Ghdqd56YyAc)

Best,
AndreyO

**Comment by jaromir-benes** [comment]
Yes, it is available in the new Model cum Plan objects. We will send you a
little tutorial on this next week.

**Comment by AndreyDO** [comment]
Thanks! Is there a solution for previous versions of Iris as well?

**Comment by jaromir-benes** [comment]
What versions exactly do you mean?

**Comment by AndreyDO** [comment]
Still stuck with 2014 and 2015 versions.

**Comment by Gleb22001** [comment]
> Yes, it is available in the new Model cum Plan objects. We will send you a
> little tutorial on this next week.
> […](#)

Hello! Is it possible to get this tutorial too?
